### PR TITLE
chore: set docker server entrypoint to dockerd to avoid TLS generation

### PR DESCRIPTION
**Pull request by andrewrynhard**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ name: default
 
 services:
   - name: docker
-    image: docker:dind
+    image: docker:19.03-dind
+    entrypoint: [dockerd]
```
Comment (bradbeam) [inline .drone.yml]: want to add 
```
command:
  - --log-level=error   
```
to suppress docker info logs?

Comment (andrewrynhard) [reply .drone.yml]: I see no harm in keeping them.

Comment (andrewrynhard) [reply .drone.yml]: Plus if we do that we have to specify the entire `command` argument.

Comment (bradbeam) [reply .drone.yml]: > I see no harm in keeping them.
 they provide no value, so it just takes up space

> Plus if we do that we have to specify the entire command argument.
shouldnt need to; we do this with talos to set the dns servers https://github.com/talos-systems/talos/blob/master/.drone.yml#L12

Comment (andrewrynhard) [reply .drone.yml]: > they provide no value, so it just takes up space

It provided value in figuring out that its generating TLS :)

**Pull request merged**